docs/user-manual/reference/bootloader.md

```diff
@@ -1,10 +1,19 @@
-# Bootloader
+# IMX Bootloader
 
-The bootloader is embedded firmware stored on the IMX and is used to update the IMX application firmware.  Updating the bootloader firmware is required occasionally when new functionality is required.
+The IMX bootloader is embedded firmware stored on the IMX and is used to update the IMX application firmware.  
+
+## Application Firmware Update
+
+The following are conditions for the IMX firmware update.
+
+- **Bootloader Enable** - At start of the firmware update process, the IMX is sent a "bootloader enable" command to reboot the processor into bootloader mode.  The IMX will not reboot into application mode until a valid firmware upload has completed.
+- **Handshake Sequence** - A handshake sequence consisting of five consecutive sync characters (`U`) spaced 10 ms (minimum 5 ms) apart is sent to the IMX to initiate a given port and close all other ports to prevent interference.
+- **UART Auto Close** - A known issue that exists in the IMX-5.0 bootloader version v5g and prior versions will disable all UARTs if no handshake sequence is received within 50 seconds of the bootloader start.  The USB port does not automatically close due to no handshake reception.
```

Comment:
That you have given this a title "UART Auto Close" suggests that its a feature, and not un-intended behavior.  I would remove the title, or replace it with a heading for "Known Issues".